mail.php

```diff
@@ -0,0 +1,66 @@
+<?php
+require 'vendor/autoload.php';
+require 'validation.php';
+use Dotenv\Dotenv;
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+/**
+ * A class used for sending email to recipient;
+ */
+class mail
+{
+  /**
+   * A function used for sending email using phpmailer package.
+   *
+   * @return void
+   */
+  public function mailSet()
+  {
+    $ob = new validation();
+    $dotenv = Dotenv::createImmutable(__DIR__);
+    $dotenv->load();
+    // Storing user name and password from .env file.
+    $user = $_ENV["username"];
+    $psw = $_ENV["password"];
+    // If email format is not valid then throws a error message.
+    if (!$ob->valid($_POST['email'])) {
+      $message = "Invalid email format.";
+      header('location:index.php?message=' . $message);
+    } 
+    else {
+      if (isset($_POST['submit'])) {
+        // Creating a object of PHPMailer class.
+        $mail = new PHPMailer(TRUE);
+        try {
+          $mail->isSMTP();
+          $mail->Host       = 'smtp.gmail.com';
```

Comment:
Why so extra spaces?
```suggestion
          $mail->Host = 'smtp.gmail.com';
```